Core/easylistprivacy.js

```diff
@@ -0,0 +1,23 @@
+//
+//  easylistprivacy.js
+//  DuckDuckGo
+//
+//  Copyright © 2017 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+var easylistData = decodeBase64("${easylist_privacy}")
+ABPFilterParser.parse(easylistData, duckduckgoBlockerData.easylist)  
```

Comment:
Should this be duckduckgoBlockerData.easyprivacylist?

Reply:
At the time no, but I've separated the two easylist checks in to two separate things.